### PR TITLE
Updated styling

### PR DESCRIPTION
**Pull request by CalebLangbroek**
Added navbar toggle for mobile
Removed margins on header
Converted header img to jpg
Added animations for sliding and fading in

### CODE REVIEW

**Comment by MykalMachon** [comment]
Hey Caleb, Thanks for the PR 🎉

I'm out right now and reviewing from my phone; but at first glance this looks awesome.

Will do a more extensive review when I get home in an hour or so 😊

**Comment by CalebLangbroek** [comment]
@MykalMachon Hey Mykal, sounds good! Let me know if you find anything you want changed. Cheers.

**Comment by MJJBennett** [comment]
This is great. The hamburger menu is a huge improvement for mobile accessibility, although the page content is still pretty unreadable in mobile (#6 should fix that, I think) this PR definitely solves #5. My only question would be if you have any suggestions for fixing the header bar in this, the original header image unfortunately wasn't very flexible and, at least on my screen, this PR stretches it a somewhat uncomfortable amount.

**Comment by CalebLangbroek** [comment]
@MJJBennett Yeah, I was planning to work on #6 next. I also noticed the stretching of the header image. I'm thinking this could be solved by either reverting back to a max width on the header or using a higher res image. Personally, I prefer headers that span the whole page, so if you have a high res header that would be great.

**Comment by MJJBennett** [comment]
@CalebLangbroek So, funny story, the way I got this header image was by taking a lower quality image, upscaling it, adding (more) blur, and then superimposing a higher quality CSA logo PNG over top of the blurred/upscaled one. That is to say, I think this is about as high quality as I can get this image.

That being said, I don't think it would be super hard to make a similar (or different) image for a new header, I just haven't really thought about it. This one does have the issue that its aspect ratio just doesn't work that well for modern website headers - as you say, headers that span the entire page are much better, and this one is just too tall to be particularly useful on widescreens.

Unrelated, but do you think it would be a good idea to slightly speed up the time it takes the page to 'fade in'? I'm not really familiar with that kind of UX/UI though.

File: src/components/layout.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ const Layout = ({ children }) => {
       <Header />
       <div className="flex-horizontal" id="header">
         <div className="img" id="header-image">
-          <img src="/images/csaheader.png" alt="CSA Header" />
+          <img src="/images/csaheader.jpg" alt="CSA Header" />
```
Comment (MykalMachon) [inline src/components/layout.js]: To make this a little prettier we can use `object:-fit: cover` in the CSS for this. 
See line 38 in main.css for the CSS selector & properties for this image!

**Comment by MykalMachon** [comment]
@MJJBennett and @CalebLangbroek regarding the bluriness of the image,
we can make a quick fix to this using the [css `objec-fit` property](https://www.w3schools.com/css/css3_object-fit.asp). It scales the image dynamically without stretching & is a good quick fix until we get a more permanent solution in place. 

See [My comment here](https://github.com/ufvcsa/ufvcsa.github.io/pull/29/files#r381573776) for where to put the rule!

**Comment by CalebLangbroek** [comment]
@MJJBennett and @MykalMachon I have another idea we can try. If you have separate images of the background and CSA logo, we could put the logo over the background using css. That way the background can expand and the logo will always stay high res and centered (I noticed when it stretches sometimes the logo isn't properly centered).

**Comment by MykalMachon** [comment]
The load in animation on page transitions looks great as well @CalebLangbroek!
My only comment being that the distance that the block travels on load-in seems a little large but besides that I think it looks great!

**Comment by MykalMachon** [comment]
> If you have separate images of the background and CSA logo, we could put the logo over the background using css

**This solution would be awesome;** a few words of caution being that I know the CSS we're currently using is pretty fragile and implementing something like this may require quite a bit of tweaking. 

_That said, **if you're willing to spend some time refactoring the CSS to get that layout to work I'm all for it!** I'm also here if you have any questions or need some support on getting it done if you decide to go forward with it._ @CalebLangbroek

**Comment by MJJBennett** [comment]
The background image actually has the logo on it, it's just way lower quality. When I upscaled it I had to do a lot of manual tweaking in gimp to get the blurry background version to be hidden properly. However it might be possible to do the math to get CSS to do that for us.

**Comment by CalebLangbroek** [comment]
@MykalMachon and @MJJBennett Okay, the animations and the header stretching should be fixed.

**Comment by MJJBennett** [comment]
Huge fan of this PR. :+1: from me.

**Comment by MykalMachon** [comment]
I'm good with it too! Thanks again @CalebLangbroek 
Merging now.

**Pull request merged**